app/code/Magento/MessageQueue/Console/RestartConsumerCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MessageQueue\Console;
+
+use Magento\Framework\Console\Cli;
+use Magento\Framework\MessageQueue\PoisonPill\PoisonPillPutInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Command for put poison pill for MessageQueue consumers.
+ */
+class RestartConsumerCommand extends Command
+{
+    private const COMMAND_QUEUE_CONSUMERS_RESTART = 'queue:consumers:restart';
+
+    /**
+     * @var PoisonPillPutInterface
+     */
+    private $poisonPillPut;
+
+    /**
+     * @param PoisonPillPutInterface $poisonPillPut
```

Comment:
Many DocBlocks of this PR violate Magento's standards (https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html)

Reply:
Thank you for your review!
May I ask you to be more specific?

Reply:
For example,

> The @inheritdoc tag SHOULD NOT be used. If a child class method requires a long description to explain its purpose, it may use @inheritdoc to indicate the new description is intended as an addition to the parent method description. In general, such method overrides are a code smell and should be used as an incentive to make the code more self-documenting if possible.
https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#inheritdoc

> The declaration of all arguments (if any) using the @param tag, unless the argument type is indicated in the method signature.
https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#functions-methods

> In general, method return type signatures should be prefered over @return type annotations.
https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#return